ccdb5_ui/templates/ccdb5_ui/ccdb-main.html

```diff
@@ -47,15 +47,43 @@
 {% endblock %}
 
 {% block page_js %}
-  <!--[if IE 9]><!-->
-  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=Intl.~locale.en"></script>
-  <!--<![endif]-->
```

Comment:
You'll want to remove this block from the other file too

Reply:
👍